### PR TITLE
Updating README.md, adding 2 missing payment methods at 'Enable payme…

### PR DESCRIPTION
**Pull request by guzzilar**
Updating document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Once the installation is done, the next thing that you are going to do is to con
 
 #### Enable payment methods
 
-> Note that Alipay and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
```
Comment (pitiphong-p) [inline README.md]: Can we swap those two options were just added?

File: README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Once the installation is done, the next thing that you are going to do is to con
...
 
 There are 3 payment methods that you can enable on your store, which are **Alipay**, **Credit / Debit Card**, and **Internet Banking**.
```
Comment (pitiphong-p) [inline README.md]: Do we need to add those payment methods here too?

File: README.md (hunk 1 above)
Comment (guzzilar) [reply README.md]: what do you mean by swap?

File: README.md (hunk 2 above)
Comment (guzzilar) [reply README.md]: 👍 let me update screenshot for this section too.

**Comment by guzzilar** [comment]
@pitiphong-p ok, done. (there are some old screenshots, I'll update again once make a new release, this one let me focus on adding missing payment methods from document 🙏 )

File: README.md (hunk 1 above)
Comment (pitiphong-p) [reply README.md]: I mean put the Installment in front of the Bill Payment

Comment (pitiphong-p) [reply README.md]: Never mind this comment. I just realized how you sort this list

Comment (guzzilar) [reply README.md]: 👍

**Comment by guzzilar** [comment]
Thanks for the review, that's pretty fast @pitiphong-p 👍

**Pull request merged**